plugins/outputs/exec/README.md

```diff
@@ -0,0 +1,24 @@
+# Exec Output Plugin
+
+Please also see: [Telegraf Output Data Formats](https://github.com/influxdata/telegraf/blob/master/docs/DATA_FORMATS_OUTPUT.md)
+
+This plugin executes external programs with serialized metrics data passed on stdin, one metric per line.  
+The external program is free to do what it wants with the data.
+
+### Configuration:
+
+```toml
+[[outputs.exec]]
+  # Shell/commands array
+  # Full command line to executable with parameters, or a glob pattern to run all matching files.
+  commands = ["/usr/local/bin/telegraf-output --config /etc/telegraf-output.conf"]
```

Comment:
We should only support a single command, having more than one in a single plugin will be problematic with the way metrics are buffered for outputs and it doesn't seem like something that needs done very frequently.

Also breaking from the way the exec input was done we should take the arguments for the single command as an array, to avoid issues with quoting and escaping which is problematic across platforms.